instance/models/instance.py

```diff
@@ -455,10 +455,8 @@ class AnsibleInstanceMixin(models.Model):
     """
     An instance that relies on Ansible to deploy its services
     """
-    ansible_source_repo_url = models.URLField(max_length=256,
-                                              default='https://github.com/edx/configuration.git')
-
-    configuration_version = models.CharField(max_length=50, default='master')
+    ansible_source_repo_url = models.URLField(max_length=256, blank=True)
+    configuration_version = models.CharField(max_length=50, blank=True)
```

Comment:
@mtyaka My reasoning for what I did was more or less identical to what you wrote above.  I've adopted the approach slightly to set defaults automatically before saving, since this is the approach @antoviaque and you prefer.